app/helpers.js

```diff
@@ -35,3 +40,12 @@ exports.changeUserIdByArtist = album => {
   delete album.userId;
   return album;
 };
+
+exports.generateToken = user => {
+  const tokenPayload = {
+    lastName: user.lastName,
+    firstName: user.firstName,
+    email: user.email
+  };
+  return jwt.encode(tokenPayload, secret_key);
+};
```

Comment:
Move jwt encryption to a service, or create a helper specifically for it

Reply:
requested changes ok